hal_st/synchronous_stm32fxxx/CMakeLists.txt

```diff
@@ -13,6 +13,8 @@ target_link_libraries(hal_st.synchronous_stm32fxxx PUBLIC
 )
 
 target_sources(hal_st.synchronous_stm32fxxx PRIVATE
+    $<$<STREQUAL:${TARGET_MCU_FAMILY},stm32wbaxx>:SynchronousAesStm.cpp>
```

Comment:
If you add peripheral definitions for AES for the other families, then you can use #ifdef HAS_PERIPHERAL_AES in the header and source, and then this does not need to be a conditional include

Reply:
Fixed.